tests/assemble.py

```diff
@@ -90,8 +107,20 @@ def test_assemble_bss_with_value():
     assert raised
 
 
+def test_assemble_global():
+    a = Assembler()
+    a.assemble(src_global)
+    assert a.symbols.has_sym('counter')
+    assert a.symbols.has_sym('internal')
+    assert a.symbols.has_sym('entry')
+
+    exported_symbols = a.symbols.export()
+    assert 'internal' not in [x[1] for x in exported_symbols]
+    assert exported_symbols == [(0, 'counter'), (2, 'entry')]
```

Comment:
well, if you compare the complete thing anyway, the previous assert is not really necessary.
could be replaced by a comment.

Reply:
I like the previous line, because I feel it makes the test more obvious in terms of what we're testing for (i.e. it has documentation value). And I prefer encoding documentation into executable code over comments (where possible), because there's no way to check whether the comments actually match what the code does.

But, it's also true that these 2 lines test for the same thing. So I would almost opt for dropping the 2nd line (the exact comparison), because that type of check should already be covered by other tests anyway. What do you think?

Reply:
Now looking at the test's code in full, I see in that I am basically testing the one vs the other, so it's very clear what is being tested even without comment. So I will drop the `assert internal in/not in...` lines.